docs/PasswordManagement.md

```diff
@@ -35,32 +38,104 @@ Note: Before the creation of any Splunk deployments in the kubernetes namespace,
 
 ## Secrets on Docker Splunk
-When Splunk Enterprise is deployed on a docker container, ansible playbooks are used to setup Splunk. Ansible playbooks interpret the environment variable SPLUNK_DEFAULTS_URL in the container as the location to read the Splunk Secret Tokens from. The tokens are used to setup Splunk Instances running on containers inside pods.
\ No newline at end of file
+When Splunk Enterprise is deployed on a docker container, ansible playbooks are used to setup Splunk. Ansible playbooks interpret the environment variable SPLUNK_DEFAULTS_URL in the container as the location to read the Splunk Secret Tokens from. The tokens are used to setup Splunk Instances running on containers inside pods.
+
+## Support for AWS IAM Role for Service Account for SmartStore Access
+
+Splunk 9.0.4 Supports Smartstore Access using AWS IAM Role for Service Account.
+
+- AWS Identity and Access Management (IAM) provides fine-grained access control where you can specify who can access which AWS service or resources, ensuring the principle of least privilege.
+- Kubernetes Pods are given an identity through a Kubernetes concept called a Kubernetes Service Account. When a Service Account is created, a JWT token is automatically created as a Kubernetes Secret. This Secret can then be mounted into Pods and used by that Service Account to authenticate to the Kubernetes API Server.
+- AWS introduced IAM Roles for Service Accounts (IRSA), leveraging AWS Identity APIs, an OpenID Connect (OIDC) identity provider, and Kubernetes Service Accounts to apply fine-grained access controls to Kubernetes pods.
+- In Kubernetes  ProjectedServiceAccountToken feature allows a fully compliant OIDC JWT token issued by the TokenRequest API of Kubernetes to be mounted into the Pod as a Projected Volume. The relevant Service Account Token Volume Projection flags are enabled by default on an EKS cluster. Therefore, fully compliant OIDC JWT Service Account tokens are being projected into each pod instead of the JWT token
+- AWS has created an identity webhook that comes preinstalled in an EKS cluster.
+- This webhook listens to create pod API calls and can inject an additional Token into splunkd pods. This webhook can also be installed into self-managed Kubernetes clusters on AWS using [this guide](https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/SELF_HOSTED_SETUP.md)
+
+Below Example explains the steps required for setting up IAM Service Account
+
+- Follow the steps defined [here to create IAM Role for Service Account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
+
+- Once the Service Account is created, make sure it is annotated with specific AIM Role. Once everything looks good, add service account to splunk custom resource. here is the example for adding it to `Standalone` instance
```

Comment:
"specific AIM Role" that is scoped with the least amount of privilege necessary for the service account. Please make sure to restrict permissions for the role.

Reply:
have added separate point based on your suggestion here, please review